docs/questions/sharing/public-links.md

```diff
@@ -47,17 +51,43 @@ This public link export option is only available for questions, not dashboards.
 
 If you want to embed the link to your dashboard or question in a simple web page or blog post, copy and paste the iframe snippet to your destination of choice.
 
-## Assigning values to filters or hiding them via the URL
+You can update the link in the `src` attribute with [public embed parameters](#public-embed-parameters) to customize the appearance of your question or dashboard.
+
+## Public embed parameters
+
+You can add hash parameters at the end of the public link in your iframe's `src` attribute to toggle appearance settings or set filter values. 
+
+Note that it's possible to find the a public link URL behind a public embed. If someone gets access to the public link URL, they can remove the hash parameters from the URL to view the original question or dashboard (without any appearance or filter settings).
+
+If you'd like to create a secure embed that prevents people from changing filter names or values, check out [signed embedding](../../embedding/signed-embedding.md).
 
-This is a bit more advanced, but if you're embedding a dashboard or question in an iframe and it has one or more filter widgets on it, you can give those filters values and even hide one or more filters by adding some options to the end of the URL. (You could also do this when just sharing a link, but note that if you do that, the person you're sharing the link with could of course directly edit the URL to change the filters' values, or to change which filters are hidden or not.)
+### Appearance
+
+| Parameter name         | Possible values                                  |
+| ---------------------- | ------------------------------------------------ |
+| bordered               | true, false                                      |
+| titled                 | true, false                                      |
+| theme                  | null, transparent, night                         |
+| font                   | [font name](../../configuring-metabase/fonts.md) |
```

Comment:
Fonts are paid, and though it might be worth mentioning here, we can probably rely on the link to communicate that.